vmm/src/cpu.rs

```diff
@@ -90,6 +92,18 @@ use crate::vm::physical_bits;
 use crate::vm_config::CpusConfig;
 use crate::{GuestMemoryMmap, CPU_MANAGER_SNAPSHOT_ID};
 
+struct SafeWrapper<T>(T);
+unsafe impl<T> Send for SafeWrapper<T> {}
+unsafe impl<T> Sync for SafeWrapper<T> {}
+
+#[cfg(feature = "kvm")]
+static STATIC_VCPUS: RwLock<Vec<(u8 /* ID*/, SafeWrapper<*mut kvm_run>)>> = RwLock::new(vec![]);
```

Comment:
I think we can use thread locals instead as follows:
```suggestion
thread_local! {
    // explicit "const" block inside the macro enables more efficient initialization
    static KVM_RUN: Cell<*mut kvm_run> = const {Cell::new(core::ptr::null_mut())};
}
```
and then we set it with the `set` method in `start_vcpu`.

Reply:
I played a bit around with this sort of thing [in this playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=3ed5cfd2c11abc545bad6e48ea49ba59)